src/web/theme/ui/atoms/Button/Button.js

```diff
@@ -24,7 +24,7 @@ Button.propTypes = {
   onClick: PropTypes.func,
   children: PropTypes.node.isRequired,
   status: PropTypes.string,
-  type: PropTypes.oneOf(["default", "invisible", "full-size"])
+  type: PropTypes.oneOf(["default", "invisible", "full-size", "add-to-cart"])
```

Comment:
That prop is so weird, the type is meant to be "submit", "button" or "reset". And it is necessary for handling form with many buttons. Having such value for this element property is a nonsense.
Is that part of fc-lite ?